platform/ios/CHANGELOG.md

```diff
@@ -2,7 +2,39 @@
 
 Mapbox welcomes participation and contributions from everyone. Please read [CONTRIBUTING.md](../../CONTRIBUTING.md) to get started.
 
+## 4.0.3 - June 22, 2018
+
 * Fixed crash in `-[MGLStyle localizeLabelsIntoLocale:]` on iOS 9.3 (attempting to access a property that was introduced in iOS 10.0) ([#12123](https://github.com/mapbox/mapbox-gl-native/pull/12123))
```

Comment:
This needs adjusting to match the log under the 4.1.0 section

Reply:
According to @friedbunny 
> In any case: the changelog is chronological, so 4.0.3 should be placed before 4.1.0.

Reply:
I meant the `Fixed crash ...` copy 😄